src/mission_companion.cpp

```diff
@@ -616,6 +619,31 @@ npc_ptr talk_function::individual_mission( const tripoint &omt_pos,
     return comp;
 }
 
+std::vector<item *> talk_function::individual_mission_give_equipment( std::vector<item *> equipment,
+        const std::string &message )
+{
+    std::vector<item *> equipment_lost;
```

Comment:
Merge error? This seems unrelated to the sidepanel.

Reply:
hm yes, it's probably a mistake from rebasing it in order to remove conflicts.